test/model-test.js

```diff
@@ -81,7 +81,7 @@ describe('AGOL Model', function () {
     })
 
     it('should call getItemMetadata to json', function (done) {
-      agol.getItem('host1', 'item1', {getMetadata: true}, function (err, json) {
```

Comment:
Oh hi @chelm 
It's just defaulting to on. Somewhere in the refactor shuffle the place where the option was set to on disappeared.